arlo-client/src/components/Audit/Setup/Contests/types.ts

```diff
@@ -1,18 +1,5 @@
-export interface IChoiceValues {
-  id?: string
-  name: string
-  numVotes: string | number
-}
-
-export interface IContestValues {
-  name: string
-  isTargeted: boolean
-  totalBallotsCast: string
-  numWinners: string
-  votesAllowed: string
-  choices: IChoiceValues[]
-}
+import { IContest } from '../../../../types'
 
-export interface IValues {
-  contests: IContestValues[]
+export interface IContests {
+  contests: IContest[]
```

Comment:
Any reason to pass around the wrapper object instead of just an array of contests? Seems like the only time the wrapper is needed is when interacting with the API, and it might be simpler to not have to wrap/unwrap everywhere else.

Reply:
@jonahkagan It's also used in the Formik form in that structure, and makes it easier for the validation schema and so on. It's a bit of a pain, but hopefully it's not too unsemantic.